src/foam/nanos/u2/navigation/SignIn.js

```diff
@@ -46,6 +47,18 @@ foam.CLASS({
       hidden: true,
       transient: true
     },
+    {
+      class: 'String',
+      name: 'username',
```

Comment:
Why is this required? Why does identifier not work?

Reply:
We want to ask user to enter username in case of Duplicate email found on the system.
https://nanopay.atlassian.net/browse/NP-7398

Reply:
Changing the field label from identifer to username on second attempt would not suffice? 
Or are you then presenting three fields?  identifier, username, password?